lib/classes/Variables.js

```diff
@@ -145,7 +148,7 @@ class Variables {
   }
 
   getValueFromSelf(variableString) {
-    let valueToPopulate = _.cloneDeep(this.service);
+    let valueToPopulate = this.service;
```

Comment:
@erikerikson @pmuens I think you guys are right. I've been doing this since v0.5 because in v0.5 we used to serialize `serverless.yml`, so we wanted to keep an immutable version of the state of the file as it was before loading, hence the cloning. We don't do that anymore in v1, but I thought I keep it this way just in case. But considering @erikerikson use case, I think we don't need that anymore.